ansible/templates/hosts

```diff
@@ -0,0 +1,13 @@
+; the first parameter in a host is the inventory_hostname which has to be
```

Comment:
if this is meant for distributed it should go into the distributed env. 
also it should end in .j2 as it is jinja2 template script that eventually becomes a hosts file.